ifj2017/tests/02_functions/tests.json

```diff
@@ -14,13 +14,13 @@
     },
     {
       "info": "Asc value",
-      "stdout": " 104",
-      "code": "scope \n print asc(!\"ahoj\", 2); \n end scope"
+      "stdout": " 111",
```

Comment:
Není pravda, zadej si na [https://www.jdoodle.com/execute-freebasic-online](url) 

/' Zajisteni zakladni kompatibility IFJ17->FreeBASIC '/

Function Length(s As String) As Integer
  Return Len(s)
End Function

Function SubStr(s as String, i As Integer, n As Integer) As String
  Return Mid(s, i, n)
End Function

/' Zde bude nasledovat program jazyka IFJ17 '/

scope
 print asc (!"ahoj", 2);
end scope

Výstup bude 104

Reply:
Bral som to zo zadania tak že druhý parameter znamená index v poli charov ale je pravda že v tomto prípade to dáva oveľa väčší zmysel keď je to brané ako pozícia v stringu. Opravené.